chaosk8s/pod/actions.py

```diff
@@ -1,22 +1,20 @@
 # -*- coding: utf-8 -*-
-import json
-import os.path
+import math
 import random
 import re
-from typing import Union
 
-from chaoslib.exceptions import FailedActivity
 from chaoslib.types import Secrets
 from kubernetes import client
 from logzero import logger
-
+from chaoslib.exceptions import ActivityFailed
```

Comment:
Actually, this one should be where it was before because it's third party to this package. But don't worry, we'll fix it once merged.